lib/00_hello.rb

```diff
@@ -1,11 +1,10 @@
 def ask_first_name
-  puts "What's you beautiful name?"
-  print "❯ "
-  return String(gets.chomp)
+  print "What's you beautiful name?\n > "
```

Comment:
```suggestion
  puts "What's you beautiful name? > "
```

Pas besoin de `\n` si tu utilise `puts`

Reply:
Dans le cas présent, c'est pour avoir le chevron sur la ligne du dessous pour indiquer que l'utilisateur peut faire une saisie

Reply:
Autant pour moi, j'avais mal lu.